docs/formats.rst

```diff
@@ -945,3 +945,18 @@ exiting translations will be removed from the file when doing so.
 When :guilabel:`Base file for new translations` is empty and file format
 supports it, empty file is created where new strings will be added once they are
 translated.
+
+The :guilabel:`Language code style` allows you to customize language code used
+in generated filenames:
+
+Default based on the file format
+   Dependent on file format, for most of them POSIX is used.
+POSIX style using hyphen as a separator
```

Comment:
The documentation here has hyphens and underscores the wrong way around. The implementation looks correct though.

Reply:
Good point, fixed in 8d1c05bf21 .